backend/src/test/scala/cromwell/backend/validation/RuntimeAttributesValidationSpec.scala

```diff
@@ -278,5 +282,81 @@ class RuntimeAttributesValidationSpec extends WordSpecLike with Matchers with Be
         case Invalid(e) => assert(e.head == "Failed to get cpu mandatory key from runtime attributes")
       }
     }
+    
+    "contain expected default values based on a config file" in {
+      val defaultVals = Map(
+        "cpu" -> CpuValidation.configDefaultWdlValue(TestConfig.allRuntimeAttrsConfig).get,
+        "failOnStderr" -> FailOnStderrValidation.configDefaultWdlValue(TestConfig.allRuntimeAttrsConfig).get,
+        "continueOnReturnCode" -> ContinueOnReturnCodeValidation.configDefaultWdlValue(TestConfig.allRuntimeAttrsConfig).get
+      )
+
+      val expectedDefaultVals = Map(
+        "cpu" -> WdlInteger(1),
+        "failOnStderr" -> WdlBoolean(false),
+        "continueOnReturnCode" -> WdlInteger(0)
+      )
+
+      defaultVals shouldBe expectedDefaultVals
+    }
+
+    "return default values as WdlString when can't be coerced to valid WdlType" in {
```

Comment:
i.e. just let the `BadDefaultAttributeValue` take the place of the magic `WdlString("known bad value")`